### ISSUE TITLE
Add button mappings in the configuration file

### ISSUE THREAD

**Issue by anxuae** [pibooth/pibooth#432]
Possibly add possibility to change the button mapping in the configuration

From @philipbarwi in #42

**Comment by orangarot** [comment]
I would also like a option of setting the two hardware buttons to two different Keys(or mouse left and right click) so i can use it on my Linux machine without gpio pins.

**Comment by anxuae** [comment]
Hi @orangarot 

You can already do it: https://pibooth.readthedocs.io/en/latest/sources/start.html#controls

P -> trigger capture event
Ctrl + E -> trigger print event

**Comment by orangarot** [comment]
Yes, but i can´t choose a second layout without the need for a third button. With the hardware buttons i can navigate with just two buttons. 
If i want to take two pictures and print them i have to press first P then Arrow Left or Right then to print Ctrl + E, i not sure whah i have to press to delete.

It would be nice if Button One could be mapped to Left arrow key and Button two to Right arrow key. It would making a custom USB Controller much easier. 

Great Software by the way, it runs flawless on an old mac mini with Linux Mint.

**Comment by anxuae** [comment]
It will be possible with pibooth v3.0.0 coming soon: https://pibooth.readthedocs.io/en/3.x/sources/start.html#controls

`C` (capture)  and `P` (print) key will simulate hardware button

**Comment by orangarot** [comment]
Thank you for your help!
